docs/README.md

```diff
@@ -1,96 +1,30 @@
-# Rogue API
-This is the Rogue API, it is used to capture activity from members.
-All `POST` and `PUT` endpoints (except v3 endpoints) require an api key (`X-DS-Rogue-API-Key`) in the header to be submitted with the request.
+# Rogue [![wercker status](https://app.wercker.com/status/518aafc1808a35e38f658c849f93630a/s/master "wercker status")](https://app.wercker.com/project/byKey/518aafc1808a35e38f658c849f93630a) [![StyleCI](https://styleci.io/repos/64166359/shield?style=flat-rounded)](https://styleci.io/repos/64166359)
 
-## Endpoints
+This is **Rogue**, the DoSomething.org user activity service. Rogue is built using [Laravel 5.5](https://laravel.com/docs/5.5) and [React](http://reactjs.com).
 
-### Web
-#### Reviews
-Functionality
----------------------------------------------- | --------------------------------------------------------
-`PUT /reviews`                                 | [Update a post's status when admin reviews the post](endpoints/reviews.md#reviews)
+Maintained by [Team Bleed](https://github.com/orgs/DoSomething/teams/team-bleed)
+- [Jen](https://github.com/ngjo) - Product Manager
+- [Shae](https://github.com/sbsmith86) - Tech Lead
+- [Katie](https://github.com/katiecrane) - Engineer
+- [Chloe](https://github.com/chloealee) - Engineer
+- [Luke](https://github.com/lkpttn) - Product Designer
+- [Dave](https://github.com/DFurnes) - Staff Engineer
 
-#### Tags
-Endpoint                                       | Functionality
----------------------------------------------- | --------------------------------------------------------
-`POST /tags`                                 | [Add or delete a post's tag](endpoints/tags.md#tags)
+If you ever have any questions about working in Rogue, please reach out to the #team-bleed slack channel.
 
-#### Posts
-Endpoint                                       | Functionality
----------------------------------------------- | --------------------------------------------------------
-`POST /posts`                           | [Create a post](endpoints/posts.md#create-a-post-and/or-create/Update-a-signup)
-`GET /posts`                            | [Get posts](endpoints/posts.md#retrieve-all-posts)
-`DELETE /posts`                         | [Delete a post](endpoints/posts.md#delete-a-post)
+### Getting Started
 
-### v1
-#### Reportbacks
-Endpoint                                       | Functionality
----------------------------------------------- | --------------------------------------------------------
-`GET /api/v1/reportbacks`                      | [Get reportbacks](endpoints/reportbacks.md#reportbacks)
+To get started with development, follow the [installation](./docs/development/installation.md) and [contributing](./docs/development/contributing.md) documentation.
 
+To get started using the API, see our [API Documentation](./docs/endpoints)
 
-### v2
```

Comment:
I'm not seeing v2 indexed in table form like this anywhere else, just making sure we don't want to keep this!

Reply:
Also it should still be there in some format in `/endpoints/*`

Reply:
sounds good!!